src/backend/views/components/sidebar.twig

```diff
@@ -6,6 +6,9 @@
   </div>
 
   <aside class="docs-sidebar__content docs-sidebar__content--invisible">
+    <span>
```

Comment:
span looks redundant

Reply:
actually, it is used for pseudo `::after` for `⌘ P` or `Ctrl P`